demos/demos_databases_apis/microsoft/kusto/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
```

Comment:
You should save the output of at least 2 cells:

* The ~first kql returning a df they can use, and ensure something like a `| head 10` or `.sample(10)`
* Public graph viz embedded iframe via plot(), or link to it via plot(render=False)